tests/shared/messages/exi_message_container.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class ExiMessageContainer:
+    message_name: str
+    json_str: str
+    description: str = ""
```

Comment:
I would vote on putting this in the messages/__init__.py or in the tests/conftest.py. What do you think @lukaslombriserdesignwerk @shalinnijel2 ?